chore: Update tracing & metrics (#431)
kubewarden/docs: docs/howtos/telemetry/20-tracing-qs.md

![Jaeger dashboard](/img/jaeger-ui-home.png "The dashboard of Jaeger")

The Jaeger collector is properly receiving the traces generated by our PolicyServer.

> For accessing the Jaeger UI, you can use create an ingress or use
> `kubectl -n jaeger port-forward service/my-open-telemetry-query 16686`
> And go to [localhost:16686](localhost:16686).

Reviewer: ```suggestion
To access the Jaeger UI, you can create an ingress or use
`kubectl -n jaeger port-forward service/my-open-telemetry-query 16686`
then go to [localhost:16686](localhost:16686).
```
